lib/inets/src/http_client/httpc.erl

```diff
@@ -700,7 +700,9 @@ Sets options to be used for subsequent requests.
       DomainDesc :: string(),
       HostName :: uri_string:uri_string().
 set_options(Options, Profile) when is_atom(Profile) orelse is_pid(Profile) ->
-    case validate_options(Options) of
+    {ok, IpFamily} = get_option(ipfamily, Profile),
```

Comment:
When httpc:set_options(...) is called without starting httpc first we get no match `{error,inets_not_started}` on these two lines. Although calling set_options without running client is unreasonable, this case should be handled.

Reply:
It is the application intets that is not started. If the inets application is started  you can set the options for the default profile (default client instance) should work fine.

```
1> inets:start().
ok
2> httpc:set_options([{ipfamily, inet6}]).
ok
3>

```

Reply:
I have misphrased my comment, sorry. The issue is now resolved, so I'll close this thread